src/main/resources/GoogleApps/GoogleAppsConfig.xml

```diff
@@ -435,9 +483,11 @@
     <property>
       <propertiesToShow/>
     </property>
+    <property>
+      <scope>WIKI</scope>
+    </property>
```

Comment:
I see that the ``configureGlobally`` property value is still present above. It won't have any effect but it's still good to remove it. You can do that by editing in object mode and clicking on the "remove deprecated properties" link.